src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -29,6 +29,7 @@ class Trials:
 
   def __init__(self):
     self.trials = []
+    environment.set_value('TRIALS_CONFIG_FILENAME', 'clusterfuzz_trials_config.json')
```

Comment:
I think the idea was to not introduce another variable. Maybe just make this a constant?

Reply:
the linter complains when we create a constant:
```
Variable name "TRIALS_CONFIG_FILENAME" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
```

Reply:
Can it be that you are trying to do that within a class? I can see other constants in the projects, e.g.:
https://github.com/google/clusterfuzz/blob/master/src/clusterfuzz/_internal/system/process_handler.py#L47

Reply:
done, added the constant outside the class.